lib/db/mysql.js

```diff
@@ -1066,7 +1116,7 @@ module.exports = function (log, error) {
       })
   }
 
-  MySql.prototype.readOneFromFirstResult = function (sql, params) {
+  MySql.prototype.readAllResults = function (sql, params) {
```

Comment:
Coming back to this code after a while, I was confused by the name "read one from first result" because as far as I could tell, it was reading multiple things.  I've renamed it to just "read all results" which seems a closer match to what it does, but I'm happy to be schooled on why that's not accurate if it isn't.